enterprise/api_client/enterprise_catalog.py

```diff
@@ -151,3 +152,15 @@ def contains_content_items(self, catalog_uuid, content_ids):
         query_params = {'course_run_ids': content_ids}
         endpoint = getattr(self.client, self.ENTERPRISE_CATALOG_ENDPOINT)(catalog_uuid)
         return endpoint.contains_content_items.get(**query_params)['contains_content_items']
+
+    @JwtLmsApiClient.refresh_token
+    def enterprise_contains_content_items(self, enterprise_uuid, content_ids):
```

Comment:
I'm not seeing any difference between this function and contains_content_items() above

Reply:
The `contains_content_items` method above is for the catalog-specific endpoint - does this particular catalog contain the content items?

The `enterprise_contains_content_items` method here is for determining if the content items exist in any of the catalogs associated with a particular enterprise customer.

The above function uses ENTERPRISE_CATALOG_ENDPOINT whereas this one uses ENTERPRISE_CUSTOMER_ENDPOINT.

Reply:
Okay cool I overlooked that. Maybe this is something that could be DRY'd up since the only difference is the endpoint being used - I'll leave it up to you 🙂